### ISSUE TITLE
pbta-histologies.tsv has inconsistent naming

### ISSUE THREAD

**Issue by jashapiro** [AlexsLemonade/OpenPBTA-analysis#889]
#### What data file(s) does this issue pertain to?

pbta-histologies.tsv

#### What release are you using?

v18

#### Put your question or report your issue here.

I was looking at `broad_histologies` related to #627 and noticed that some histologies are listed twice when summarizing. For example, we have both `Embryonal Tumor` and `Embryonal tumor`. We also have both `Ependymal Tumor` and `Ependymal tumor`. Many scripts are resistant to these kinds of variation, it is something we should avoid! I think we had mostly standardized on sentence case for these labels: capitalize the first word and abbreviations only. 

I am not sure where these snuck in (perhaps in integrating subtyping info?) but they should be corrected before v18 release.

Some of these also affect `short_histology`, where we also have some other variation, such as `EPN` and `Ependymoma` that need to be fixed before release.

Tagging @jharenza and @kgaonkar6 to investigate.

**Comment by jharenza** [comment]
Ahhhh, thanks for catching this. Yes, we can fix this - I think it is coming from the integrate step.

**Comment by kgaonkar6** [comment]
The `EPN` and  `Ependymoma` in short_histology and `Ependymal Tumor` and `Ependymal tumor` discrepancy can be fixed with #860 where I'm adding the WGS only EPN samples in the molecular-subtyping-EPN module that are missed in the master. Should  #860 be merged to master so that we include these samples in subtyping for v18 histologies file?

For I think the only fix is changing the base histologies file because the `Embryonal Tumor` only gets added to Medullo samples in https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/57e947a657eced05a6f346e01bbbae96cad4f78e/analyses/molecular-subtyping-pathology/01-compile-subtyping-results.Rmd#L265-L283. All other Embryonal tumors have `Embryonal tumor` in broad_histology from in base histologies file.

**Comment by jaclyn-taroni** [comment]
I checked out the branch for the v18 release (#857), ran the data download script to obtain the v18 data. Looking at the `broad_histology` column of `pbta-histologies.tsv`:

```R
> unique(histologies_df$broad_histology)
 [1] "Ependymal tumor"                              
 [2] NA                                             
 [3] "Diffuse astrocytic and oligodendroglial tumor"
 [4] "Low-grade astrocytic tumor"                   
 [5] "Mesenchymal non-meningothelial tumor"         
 [6] "Tumors of sellar region"                      
 [7] "Embryonal tumor"                              
 [8] "Tumor of cranial and paraspinal nerves"       
 [9] "Benign tumor"                                 
[10] "Histiocytic tumor"                            
[11] "Choroid plexus tumor"                         
[12] "Lymphoma"                                     
[13] "Meningioma"                                   
[14] "Germ cell tumor"                              
[15] "Neuronal and mixed neuronal-glial tumor"      
[16] "Metastatic tumors"                            
[17] "Other astrocytic tumor"                       
[18] "Pre-cancerous lesion"                         
[19] "Tumor of pineal region"                       
[20] "Non-tumor"                                    
[21] "Non-CNS tumor"                                
[22] "Other tumor"                                  
[23] "Melanocytic tumor"  
```

Where the NA are for normal samples:

```R
> histologies_df %>% filter(is.na(broad_histology)) %>% count(sample_type)
# A tibble: 1 x 2
  sample_type     n
  <chr>       <int>
1 Normal        833
```

And for `short_histology`:

```R
> unique(histologies_df$short_histology)
 [1] "Ependymoma"                   
 [2] NA                             
 [3] "HGAT"                         
 [4] "LGAT"                         
 [5] "Sarcoma"                      
 [6] "Craniopharyngioma"            
 [7] "EWS"                          
 [8] "Medulloblastoma"              
 [9] "Schwannoma"                   
[10] "Embryonal tumor"              
[11] "ATRT"                         
[12] "Choroid plexus tumor"         
[13] "Neurofibroma"                 
[14] "Langerhans cell histiocytosis"
[15] "Ganglioglioma"                
[16] "CNS lymphoma"                 
[17] "Meningioma"                   
[18] "Teratoma"                     
[19] "DNET"                         
[20] "Hemangioblastoma"             
[21] "Adenoma"                      
[22] "Metastases"                   
[23] "SEGA"                         
[24] "Dysplasia"                    
[25] "Glial-neuronal tumor NOS"     
[26] "MPNST"                        
[27] "Pineoblastoma"                
[28] "CNS RMS"                      
[29] "Neuroblastoma"                
[30] "Chordoma"                     
[31] "ETMR"                         
[32] "Germinoma"                    
[33] "Neurocytoma"                  
[34] "Oligodendroglioma"            
[35] "Cavernoma"                    
[36] "Germ cell tumor"              
[37] "Rosai-Dorfman disease"        
[38] "Benign tumor"                 
[39] "Other"                        
[40] "JXG"                          
[41] "Xanthogranuloma"              
[42] "Melanocytic tumor"            
[43] "Fibromyxoid lesion"           
[44] "Myofibroblastoma"    

> histologies_df %>% filter(is.na(short_histology)) %>% count(sample_type)
# A tibble: 1 x 2
  sample_type     n
  <chr>       <int>
1 Normal        833
```

Closing this issue.
